src/pump/_license.py

```
@@ -10,27 +11,27 @@ class licenses:
     validate_table = [
         ["license_definition", {
             "compare": ["name", "confirmation", "required_info"],
-            "db": "clarin-utilities",
+            "db": allowed_db.DB_UTILITIES,
```

Pylint (3.3.7)

[error] Class 'allowed_db' has no 'DB_UTILITIES' member

(E1101)